src/index.html

```diff
@@ -8,5 +8,51 @@
   </head>
   <body>
     <h1>Calendar</h1>
+
+    <div class="calendar">
```

Comment:
- This tasks are not implemented 
```suggestion
    <div class="calendar calendar--start-day-sun calendar--month-length-29">
```